src/config.py

```diff
@@ -0,0 +1,2 @@
+KOLIBRI_PORT = 8080
```

Comment:
I'm not personally attached to it, and actually would prefer that we move over to the approach you're working on at https://github.com/learningequality/kolibri/pull/7714. This is the port Android was using so I just kept it during the migration.

Reply:
Can switch it back to 5000 or some other port in the meantime as well.

Reply:
I think one option would be to trying one or more default ports first, that Kolibri is aware of, and then choosing another random port only if those fail. This would allow things to work except in the case of "all default ports used + zeroconf detection not working" case, which should hopefully be fairly uncommon.

Reply:
Yeah, that could be a good compromise!

Reply:
I've switched the port to 5000, and will add the ability to check for open ports in a subsequent PR.

Reply:
Filed an issue here: https://github.com/learningequality/kolibri-installer-mac/issues/74